spec/services/weights_and_measures_spec.rb

```diff
@@ -90,4 +90,12 @@
       end
     end
   end
+
+  describe "UNITS" do
+    include Spree::Admin::GeneralSettingsHelper
+    it "should include all the available untis" do
+      units = WeightsAndMeasures::UNITS.values.flat_map(&:values).pluck("name").sort.uniq
+      expect(units).to eq(all_units.sort.uniq)
```

Comment:
If this is the same, shall we call `WeightsAndMeasures::UNITS.values.flat_map(&:values).pluck("name")` from `all_units`? Then we don't need to maintain two lists. We just need to make sure that the units are in the right order.